PlanetTerp.py

```diff
@@ -54,21 +54,19 @@ def professor(self, name, reviews = None):
     # reviews	boolean	Optional. Show reviews for the professors. Default: false
     # limit	    integer	Optional. Maximum number of records to return. Must be between 1 and 1000. Default: 100
     # offset	integer	Optional. Number of records to skip for pagination. Default: 0
-    def all_professors(self, type_ = None, reviews = None, limit = None, offset = None):
+    def all_professors(self, type_ = None, reviews = False, limit = None, offset = None):
         request_string = self.url + 'professors?'
         if type_:
             request_string += 'type=' + type_ + '&'
         if reviews:
-            request_string += 'reviews=' + reviews + '&'
+            request_string += 'reviews=true' + '&'
         if limit:
-            request_string += 'limit=' + limit + '&'
+            request_string += 'limit=' + str(limit) + '&'
         if offset:
-            request_string += 'offset=' + offset + '&'
+            request_string += 'offset=' + str(offset) + '&'
 
         # We ignore the last character as it will be an extra '&'
-
-        data = requests.get(request_string[:-1]).json()
-        return data
+        return requests.get(request_string[:-1]).json()
```

Comment:
Should we split this into two lines for clarity? 

Like this:
```
request_url = request_string[:-1]
return requests.get(request_url).json()
```

Reply:
We can.